scripts/setup_fvp_optee.sh

```diff
@@ -180,6 +180,8 @@ export CROSS_COMPILE=$DST_AARCH64_NONE_GCC/bin/aarch64-none-elf-
 cd $DST_KERNEL
 
 if [ ! -f ".config" ]; then
+	sed -i '/config ARM64$/a\
+	select DMA_SHARED_BUFFER' arch/arm/Kconfig
```

Comment:
Well, you're right, my suggestion won't work. DMA_SHARED_BUFFER is a hidden configuration value (it has no "prompt" text in KConfig), so you can only enable it with "select" as you are suggesting. Normally it would be the driver that selects DMA_SHARED_BUFFER, but since it is out-of-tree this is not possible.